app/assets/stylesheets/styles/ui/inputs/s3_upload.css.scss

```diff
@@ -2,6 +2,54 @@
  * A class that covers all the s3 upload buttons.
  */
 
+.thumbnails {
+  margin: 0 .5rem .5rem 0;
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  transition: .2s opacity, .2s background-color;
+
+  &:before {
+    content: '\f1f8';
+    font-family: 'FontAwesome';
+    position: absolute;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.